azure-pipelines.yml

```diff
@@ -250,7 +246,7 @@ jobs:
   - task: VSBuild@1
     name: ''
     displayName: Sign Files
-    condition: and(succeeded(), ne(variables['Build.Reason'], 'PullRequest'))
+    condition: and(succeeded(), eq(variables['RunTests'], 'true'))
```

Comment:
💡☕️ Question: Where will this variable be getting set please?

Reply:
It is set in variables tab , here is screenshot of it. 
<img width="852" alt="image" src="https://user-images.githubusercontent.com/105889062/209965738-764557b7-f6fd-4436-a006-897b235d95b5.png">

Reply:
I moved them from variables tab to variables in the pipeline definition, hopefully that clarifies things.